### DISCUSSION TITLE
Keycloak user page not loading

### DISCUSSION THREAD

**Discussion by mepradeep20** [keycloak/keycloak#28434] [Q&A]
While loading the Keyloack User screen from external system, no details loaded in the screen,
URL - https://<Host_Details>/auth/admin/<realm_name>/console/#/realms/<realm_name>/users

observed below exception in the console and no other error,

keycloak.js?version=ld7le:1333 Refused to frame 'https://<Host_Details>/auth/realms/bankingcloud/protocol/openid-connect/3p-cookies/step1.html?version=ld7le' because it violates the following Content Security Policy directive: "frame-src 'none'".

**Comment by justin-tay** [comment]
I think you are attempting to load Keycloak in an `iframe` from your main page. However when your main page was served, the `Content-Security-Policy` header has the directive `frame-src 'none'` which will prevent any `iframe` from being loaded. 

You will likely need to configure your web server to return something like `Content-Security-Policy: frame-src https://<Host_Details>`
